main.py

```diff
@@ -45,12 +53,22 @@ def main() -> int:
 
     camera = CameraDevice(0, 100, str(LOG_NAME))
 
+    # TCP Connection, do we want UDP instead (socket.SOCK_DGRAM)?
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.connect(SOCKET_HOST, SOCKET_PORT)
```

Comment:
Create a class that holds the socket and handles all the details of connecting. Add a send() method.